doc/dev/index.rst

```diff
@@ -1337,21 +1337,21 @@ manually.
 However, there are times when the automated tests do not cover a particular
 scenario and manual testing is desired. It turns out that it is simple to
 adapt a test to stop and wait after the Ceph installation phase, and the
-engineer can then ssh into the running cluster.
+engineer can then ssh into the running cluster. Simply add the following
+snippet in the desired place within the test YAML and schedule a run with the
+test::
 
-This common use case is currently provided for by the following command::
+    - exec:
+        client.0:
+          - sleep 1000000000 # forever
```

Comment:
@tchaikov Good idea! I read the documentation at https://github.com/ceph/teuthology/blob/master/teuthology/task/interactive.py#L13-L29 from which I suppose that this causes an interactive Python shell to be run, but I'm not sure how to connect to that shell?